### PR TITLE
Sentiment analysis of POTUS tweets using R

### PR DESCRIPTION
**Pull request by kwichmann**
I used the AFINN-111 data set to do a simple sentiment analysis of the tweets. The code is in a Rmd file, and the knitr'ed results are included as pdf.

Much more could obviously be done with this, but this is a start.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 config.js
 node_modules/
 .DS_Store
+.RData
```
Comment (shiffman) [inline .gitignore]: Should this be `*.RData` and `*.Rhistory`?

**Review by shiffman — CHANGES_REQUESTED** [review]
Are the files in `sentiment_cache` and `sentiment_files` necessary?

File: .gitignore (hunk 1 above)
Comment (kwichmann) [reply .gitignore]: Ah yes, of course.

**Comment by kwichmann** [comment]
Removed now.

**Review by shiffman — APPROVED** [review]

**Comment by shiffman** [comment]
fantastic, thanks!

**Pull request merged**